benchmarks/quant-hela-method-comparison/scripts/04_compute_metrics.py

```
@@ -0,0 +1,581 @@
+        rank_matrix[dataset_id] = ranks
+
+    # Compute Spearman correlation matrix
+    corr_matrix = rank_matrix.corr(method="spearman")
+
+    return corr_matrix
+
+
+def compute_expression_stability(
+    datasets: Dict[str, DatasetInfo],
+    method: str,
+    proteins: List[str] = None,
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)